packages/reassure-compare/src/output/console.ts

```diff
@@ -24,6 +27,11 @@ export function printToConsole(data: CompareResult) {
   console.log('');
 }
 
+function printMetadata(entry: Metadata) {
+  console.log(
+    ` - baseline: ${entry.baseline?.branch} :: ${entry.baseline?.commitHash} | current: ${entry.current?.branch} :: ${entry.current?.commitHash}`
+  );
+}
```

Comment:
I would rather not assume that we have either piece of metadata, and do something like below:

```suggestion
function buildMetadata(name: string, metadata?: MeasurementMetadata) {
  if (metadata.branch && metadata.commitHash) {
     return `**${name}**: \`${metadata.branch}\` (\`${metadata.commitHash}\`)`;
  }

  if (metadata.branch) {
    return `**\`${name}**\`: ${\`metadata.branch\`}`;
  }

  if (metadata.branch) {
    return `**${name}**: \`${metadata.commitHash}\``;
  }

  return `**${name}**: missing metadata`;
}
```